UnityProject/Assets/Scripts/UI/IngameMenu/GUI_IngameMenu.cs

```diff
@@ -29,30 +29,6 @@ void Awake()
 		}
```

Comment:
This class derives from NetworkBehaviour but it is on a child object of managers.prefab. I'll be converting this back to a monobehaviour with a follow up fix